src/main/java/io/github/oliviercailloux/y2018/apartments/valuefunction/Criterion.java

```diff
@@ -1,14 +1,95 @@
 package io.github.oliviercailloux.y2018.apartments.valuefunction;
 
 public enum Criterion {
-  TELE,
-  TERRACE,
-  WIFI,
-  FLOOR_AREA,
-  FLOOR_AREA_TERRACE,
-  NB_BATHROOMS,
-  NB_BEDROOMS,
-  NB_SLEEPING,
-  NB_MIN_NIGHT,
-  PRICE_PER_NIGHT;
+  TELE(ValueFunctionType.IS_BOOLEAN_CRESCENT),
+  TERRACE(ValueFunctionType.IS_BOOLEAN_CRESCENT),
+  WIFI(ValueFunctionType.IS_BOOLEAN_CRESCENT),
+  FLOOR_AREA(ValueFunctionType.IS_NOT_BOOLEAN_CRESCENT),
+  FLOOR_AREA_TERRACE(ValueFunctionType.IS_NOT_BOOLEAN_CRESCENT),
+  NB_BATHROOMS(ValueFunctionType.IS_NOT_BOOLEAN_CRESCENT),
+  NB_BEDROOMS(ValueFunctionType.IS_NOT_BOOLEAN_CRESCENT),
+  NB_SLEEPING(ValueFunctionType.IS_NOT_BOOLEAN_CRESCENT),
+  NB_MIN_NIGHT(ValueFunctionType.IS_NOT_BOOLEAN_DECREASE),
+  PRICE_PER_NIGHT(ValueFunctionType.IS_NOT_BOOLEAN_DECREASE);
+
+  private ValueFunctionType valueFunctionType;
+
+  Criterion(ValueFunctionType valueFunctionType) {
+    this.valueFunctionType = valueFunctionType;
+  }
+
+  /**
+   * Permet de récupérer le type associé
```

Comment:
Merci de traduire cette partie en anglais

Reply:
Fix in 21da4d57b892d89688eb0758884f7e53e140851d